Vagrantfile

```diff
@@ -104,49 +104,66 @@ Vagrant.configure(2) do |config|
     mount -t overlayfs overlayfs -o lowerdir=#{overlay_lower},upperdir=#{overlay_upper} #{overlay_mount}
   SCRIPT
 
-  # run install script
-  plugin_string = plugins.join(" ")
-  config.vm.provision "shell", inline: <<-SCRIPT
-    if [ ! -f /var/local/reddit_installed ]; then
-      echo "running install script"
-      cd /home/#{vagrant_user}/src/reddit
-      REDDIT_PLUGINS="#{plugin_string}" REDDIT_DOMAIN="#{hostname}" ./install-reddit.sh
-      touch /var/local/reddit_installed
-    else
-      echo "install script already run"
-    fi
-  SCRIPT
-
-  # setup private code
-  if File.exist?("#{code_share_host_path}/private/vagrant_setup.sh")
-    config.vm.provision "shell",
-      path: "#{code_share_host_path}/private/vagrant_setup.sh",
-      args: [vagrant_user]
+  config.vm.define "travis" do |travis|
+      # run install script
+      plugin_string = plugins.join(" ")
+      travis.vm.provision "shell", inline: <<-SCRIPT
+        if [ ! -f /var/local/reddit_installed ]; then
+          echo "running install script"
+          cd /home/#{vagrant_user}/src/reddit
+          ./install/travis.sh vagrant
+          touch /var/local/reddit_installed
+        else
+          echo "install script already run"
+        fi
+      SCRIPT
   end
 
-  # inject test data
-  config.vm.provision "shell", inline: <<-SCRIPT
-    if [ ! -f /var/local/test_data_injected ]; then
-      cd /home/#{vagrant_user}/src/reddit
-      sudo -u #{vagrant_user} reddit-run scripts/inject_test_data.py -c 'inject_test_data()'
-      touch /var/local/test_data_injected
-    else
-      echo "inject test data already run"
-    fi
-
-    # HACK: stop and start everything (otherwise sometimes there's an issue with
-    # ports being in use?)
-    reddit-stop
-    reddit-start
-  SCRIPT
-
-  # additional setup
-  config.vm.provision "shell", inline: <<-SCRIPT
-    if [ ! -f /var/local/additional_setup ]; then
-      apt-get install -y ipython avahi-daemon
-      touch /var/local/additional_setup
-    else
-      echo "additional setup already run"
-    fi
-  SCRIPT
+  config.vm.define "default" do |redditlocal|
```

Comment:
note to self: will need to do `vagrant up default` to name the VM and ensure that this block runs